### PR TITLE
Adding script to migrate users extendedAt values to the keys they own

### PR DESCRIPTION
**Pull request by rowannekabalan**
## What does this change?
Honors key extensions of users who extended their keys based on the old extension logic

### CODE REVIEW

File: scripts/migrate-extendedAt-dates.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+items = response['Items']
+for item in items:
+    id = item['id']
+    extendedAt = item['extendedAt']
+    print(f"Item {id} has extendedAt date {extendedAt}")
+    keysToExtend = keysTable..scan(
+          FilterExpression=Attr('bonoboId').eq(id),
+    )
+    print(f"Found {keysToExtend['Count']} keys / scanned {extendedUsers['ScannedCount']}")
+    for key in keys:
```
Comment (JustinPinner) [inline scripts/migrate-extendedAt-dates.py:37]: Should this be `for key in keysToExtend`?

File: scripts/migrate-extendedAt-dates.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+)
+
+print(f"Found {extendedUsers['Count']} users / scanned {extendedUsers['ScannedCount']}")
+
+items = response['Items']
+for item in items:
+    id = item['id']
+    extendedAt = item['extendedAt']
+    print(f"Item {id} has extendedAt date {extendedAt}")
+    keysToExtend = keysTable..scan(
```
[the remark this answers is not in the document]
Comment (rowannekabalan) [reply scripts/migrate-extendedAt-dates.py]: Fixed

File: scripts/migrate-extendedAt-dates.py (hunk 1 above)
Comment (rowannekabalan) [reply scripts/migrate-extendedAt-dates.py]: Ah of course! Fixed as well

**Review by JustinPinner — APPROVED** [review]
Looks pretty solid to me 👍

**Pull request merged**